extras/container/heketi-start.sh

```diff
@@ -113,5 +181,5 @@ if [[ "$(stat -c %s ${HEKETI_PATH}/heketi.db)" == 0 && -n "${HEKETI_TOPOLOGY_FIL
     fg %1
 else
     # just start in the foreground
-    exec /usr/bin/heketi --config=/etc/heketi/heketi.json
+    exec "$HEKETI_BIN" --config=/etc/heketi/heketi.json
```

Comment:
Same here, having " around an executable looks weird. Just use this?

    exec ${HEKETI_BIN} --config=/etc/heketi/heketi.json